### ISSUE TITLE
cf ssh broken for docker containers that don't expose a port

### ISSUE THREAD

**Issue by x6j8x** [cloudfoundry/cloud_controller_ng#883]
Thanks for submitting an issue to `cloud_controller_ng`. We are always trying to improve! To help us, please fill out the following template.

## Issue

When pushing a docker container that doesn't expose a port we're no longer able to `cf ssh` into it (CF v271).

```bash
$ cf ssh ubuntu
FAILED
Error: SSH session allocation failed: EOF
```

Inspecting the desired-lrp data in bbs via cfdot, we observed that docker based LRPs no longer have port 2222 in their ports list (containers pushed prior to the update to v271 contain this port) .

We were running CF v268 before.

**Comment by aashah** [comment]
@x6j8x Do you have the `temporary_local_*` flags enabled?

**Comment by x6j8x** [comment]
Yes, we have the temporary_local_* flags enabled, but we can't say for sure if the problem started before or after enabling them (we first updated to cf v271 and then several hours later enabled the temporary_local_* flags).

**Comment by aashah** [comment]
Could you also check:

1. The `allow_app_ssh_access` manifest property.
1. The `default_app_ssh_access` manifest property.
1. Whether the space allows SSH (`cf space-ssh-allowed SPACE_NAME`)

**Comment by x6j8x** [comment]
`allow_app_ssh_access: true`
`default_app_ssh_access` not set

```
$ cf space-ssh-allowed acceptance
ssh support is enabled in space 'acceptance'
```

**Comment by eruvanos** [comment]
Hello @aashah,
any news about this issue?

**Comment by x6j8x** [comment]
@aashah Short update from my side... We've updated to Cloudfoundry v272 and the problem still exists.

**Comment by zrob** [comment]
Resolved in 9517d85.  Should be available in upcoming capi-release 1.42.0
